hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractClusterWideIterator.java

```diff
@@ -83,7 +83,7 @@
     protected final int partitionCount;
     protected int partitionIndex = -1;
 
-    protected int lastTableIndex;
+    protected int lastTableIndex = Integer.MAX_VALUE;
```

Comment:
can you document the reason of this assignment `lastTableIndex = Integer.MAX_VALUE`?